FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2SegmentedControlActivity.kt

```diff
@@ -198,6 +203,7 @@ class V2SegmentedControlActivity : DemoActivity() {
                                         selectedTab = idx
                                     },
                                     enabled = enableTabs,
+                                    notificationDot = Random.nextBoolean()
```

Comment:
remove the random value usage as it shows deterministic behaviour

Reply:
done